ts/state/createStore.ts

```diff
@@ -25,7 +25,8 @@ const logger = createLogger({
 });
 
 // Exclude logger if we're in production mode
-const middlewareList = env === 'production' ? [promise] : [promise, logger];
+const enableLogging = false && env === 'production'; // ALWAYS TURNED OFF
+const middlewareList = enableLogging ? [promise] : [promise, logger];
```

Comment:
Won't this always *enable* the logging?
(I.e. if `enableLogging` is false then `middlewareList = [promise, logger];`?

Reply:
😅 oops

Reply:
```suggestion
const middlewareList = !enableLogging ? [promise] : [promise, logger];
```